assets/js/pages/ClusterDetails/HanaClusterDetails.test.jsx

```diff
@@ -226,6 +226,42 @@ describe('HanaClusterDetails component', () => {
     );
   });
 
+  it('should display the cluster maintenance mode', async () => {
+    const {
+      clusterID,
+      clusterName,
+      cib_last_written: cibLastWritten,
+      type: clusterType,
+      sid,
+      provider,
+      details,
+    } = clusterFactory.build();
```

Comment:
Would it be possible to have here maintenance as true?
As false is the default option, and it is tested somewhere else.
`clusterFactory.build({ details: {maintenance_mode: true}})` would make it I think

Reply:
done!